src/components/formik-forms/formik-select.jsx

```diff
@@ -0,0 +1,62 @@
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+import {Field} from 'formik';
+
+const ValidationMessage = require('../forms/validation-message.jsx');
+
+require('../forms/input.scss');
+require('../forms/row.scss');
+
+const FormikSelect = ({
+    className,
+    error,
+    options,
+    validationClassName,
+    ...props
+}) => {
+    const optionsList = options.map((item, index) => (
+        <option
+            key={index}
+            value={item.value}
+        >
+            {item.label}
+        </option>
+    ));
+    return (
+        <div className="col-sm-9 row row-relative">
+            <Field
+                className={classNames(
+                    'select',
+                    className
+                )}
+                component="select"
+                {...props}
+            >
+                {optionsList}
+            </Field>
+            {error && (
+                <ValidationMessage
+                    className={validationClassName}
+                    message={error}
+                />
+            )}
+        </div>
+    );
+};
+
+
+FormikSelect.propTypes = {
+    className: PropTypes.string,
+    error: PropTypes.string,
+    // expect structure like: [
+    //    {value: 'US', label: 'United States'}
+    //    {value: 'AG', label: 'Angola'}
+    //    ...
+    // ]
+    options: PropTypes.arrayOf(PropTypes.any),
```

Comment:
If we have specific requirements, about the object shape, I think we should write a more specific validator. I tried the code below out and it seems to work as I hoped:
options: PropTypes.arrayOf(PropTypes.shape({
     value: PropTypes.string.isRequired,
     label: PropTypes.string.isRequired,
 })),

Reply:
Good idea, thanks!